Assets/Mirror/Examples/ListServer/ListServer.cs

```diff
@@ -38,6 +38,7 @@ public ServerStatus(string ip, /*ushort port,*/ string title, ushort players, us
     public class ListServer : MonoBehaviour
     {
         [Header("List Server Connection")]
+        public NetworkManager manager;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/paulpach/MirrorNG/pullRequest?prid=4841467)